### DISCUSSION TITLE
CSV export - download all top pages

### DISCUSSION THREAD

**Discussion by RobertJoonas** [plausible/analytics#1521] [Anything Else]
Currently, when using the dashboard CSV export tool, there is a limit of 100 on the top pages report. I would like a simple way of downloading all that data without programming and using the Stats API.

Would help if I could set the limit myself when starting the download.

(feedback via email)

**Comment by metmarkosaric** [comment]
We’ve improved our CSV export functionality. You can still export any segment of your audience directly from the dashboard and now you can also perform a full export from the site settings without any limits on the number of entries or pages. See more here: https://plausible.io/docs/export-stats
